### ISSUE TITLE
MSI interrupt enabler needed for 1080 ti passthrough

### ISSUE THREAD

**Issue by sanjay900** [joeknock90/Single-GPU-Passthrough#68]
### Hardware

- GPU: Nvidia 1080 TI
- CPU: Intel(R) Core(TM) i7-6800K CPU @ 3.40GHz

### Files

- XML: [win10.xml.txt](https://github.com/joeknock90/Single-GPU-Passthrough/files/6679811/win10.xml.txt)
- start.sh: [start.sh.txt](https://github.com/joeknock90/Single-GPU-Passthrough/files/6679813/start.sh.txt)
- revert.sh: [revert.sh.txt](https://github.com/joeknock90/Single-GPU-Passthrough/files/6679816/revert.sh.txt)

I tried this multiple times with different scripts, and eventually worked out that for my 1080 TI, it was required to grab a ROM file for my GPU, and then patch it. After this, i would see the bootloader, but windows would always crash before booting. Eventually, i worked out that i needed to run [MSIInterruptEnabler](https://github.com/TechtonicSoftware/MSIInturruptEnabler). Once this was installed, my passthrough worked perfectly, however it was difficult to work out that this was necessary, and i eventually found the hint on a random comment somewhere. 

I'm not really sure where this information belongs, but it does seem like it could be useful to someone, as it was super difficult to debug.

**Comment by awoimbee** [comment]
Hi, do you have more information about the ROM file thing ?
The README states [here](https://github.com/joeknock90/Single-GPU-Passthrough/blob/master/README.md#matoking) that it's not needed (anymore), but I can't get it to work on this might be why (I also have a 1080TI).
To quote the current version of the README:
```
Patching the rom is no longer required but I never would have written this guide without the original work
```

**Comment by dArkyunn** [comment]
@awoimbee Apparently, ROM patching is not needed on newer Nvidia drivers, since Nvidia started supporting VMs. That information, too, was taken from a random Reddit comment though, so might need to check this information.

**Comment by sanjay900** [comment]
I found that I needed rom patching otherwise I wasn't even able to see a
boot screen.

**Comment by sanjay900** [comment]
@awoimbee out of curiousity, are you able to see the tianocore startup screen when you use your 1080ti? i found that i didnt get a startup screen at all without the rom patching, and then it would crash after loading the nvidia driver before making it to the desktop without the msi interrup enabler

**Comment by awoimbee** [comment]
In order to see the tianocore startup screen I need to use a patched founder edition bios (I didn't manage to patch the bios of my 1080Ti PNY OC something something).
I don't need MSIInterruptEnabler tho.
BTW my issue when I posted my initial comment was that I misconfigured the PCI bus & function number of the nvidia audio and nvidia gpu devices (both need to be on the same bus but different function, whatever that means :p ).
